templates/k8smetricsservicemonitor.go

```diff
@@ -30,6 +30,9 @@ var params = map[string][]string{
 		"{__name__='node_disk_write_time_seconds_total'}",
 		"{__name__='node_disk_reads_completed_total'}",
 		"{__name__='node_disk_writes_completed_total'}",
+		"{__name__='cluster:kubelet_volume_stats_used_bytes:provisioner:sum'}",
+		"{__name__='cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum'}",
+		"{__name__='job:kube_pv:count'}",
```

Comment:
kube_pv:count was not present, rest were

Reply:
So why are we federating this metric?

Reply:
I checked up, this metric doesn't come from in-cluster monitoring but  from ocs-metrics.
It counts the number of pv's which are provisioned by ocs storageclasses, since I was testing on a provider cluster no pv was created, this metric would be available for consumer clusters. Adding it here won't break anything in provider deployed clusters.We would still require them for consumer clusters.

Reply:
https://github.com/red-hat-storage/ocs-operator/blob/fc97cf4ee0778bedab67250227ce489f6d3cf4f5/controllers/storagecluster/prometheus/localcephrules.yaml#L26

Reply:
will remove this in the next patch